pkg/maintenance/release/release.go

```diff
@@ -23,6 +24,7 @@ const (
 	DefaultServiceIDLabel = "metadata.appcat.vshn.io/serviceID"
 	RevisionLabel         = "metadata.appcat.vshn.io/revision"
 	AutoUpdate            = "metadata.appcat.vshn.io/autoUpdate"
+	MinimumRevisionAge    = 7 * 24 * time.Hour // 1 week
```

Comment:
Just one nit: instead of hardcoding this, maybe have it configurable via the component.